### DISCUSSION TITLE
YLSIdeas's Feature Flags for Laravel

### DISCUSSION THREAD

**Discussion by peterfox** [HassanZahirnia/laravel-package-ocean#100] [Package Suggestions]
I maintain this package. It's pretty popular at 500+ stars and 500K+ installs. Would be neat to see it on the site https://github.com/ylsideas/feature-flags.

Would likely fit best into the dev tools or utilities list.

**Comment by HassanZahirnia** [comment]
Hey @peterfox ! 😊👋🏻

Your package already exist in the database!

**Reply by peterfox** [reply]
Hi @HassanZahirnia awesome! I wasn't aware as I had a quick look but couldn't see it and couldn't see any discussions about it being added. Thank you 😁
